client/src/info_bar.rs

```diff
@@ -68,12 +68,22 @@ pub fn InfoBar<'a>(cx: Scope<'a, InfoBarProps<'a>>) -> Element<'a> {
                 class: "moves-container",
                 style: "position: relative; overflow-y: auto;",
 
-                "Moves:"
+                p { "Rounds:" }
                 cx.props.game.with(|game| {
-                    game.get_move_history().into_iter().enumerate().map(|(i, turn_str)| {
+                    game.get_rounds_str().into_iter().enumerate().map(|(i, (mv1, mv2))| {
                         rsx! {
-                            p {
-                                "{i + 1} {turn_str}"
+                            div {
+                                style: "margin-bottom:15px;",
+                                tr {
+                                    td {
+                                        style: "padding-right: 15px;" ,
+                                        "{i + 1}." }
+                                    td {
+                                        style: "padding-right: 15px;",
+                                        "{mv1}"
+                                    }
+                                    td { "{mv2}" }
```

Comment:
Maybe do a `padding-right` on this one too, so if there is something to the right of the info bar later, the table cells have the same padding.